gammapy/datasets/tests/test_flux_points.py

```diff
@@ -102,6 +102,19 @@ def test_flux_point_dataset_flux_pred(dataset):
     assert_allclose(dataset.flux_pred()[0].value, 0.000472, rtol=1e-3)
 
 
+@requires_data()
+def test_flux_point_dataset_stat(dataset):
+    dataset.stat_type = "chi2"
+    fit = Fit()
+    fit.run([dataset])
+    assert_allclose(dataset.stat_sum(), 25.205933, rtol=1e-3)
+
+    dataset.stat_type = "distrib"
```

Comment:
There is no test here with "profile"?

Reply:
`test_flux_points` read a flux points objects for which stat_can is not computed so the test for profile can not be done there and is done in `test_lightcurve` instead.